### ISSUE TITLE
Editor disabled buttons are not focusable

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#6813]
Editor disabled buttons are not focusable. This is in line with the default behavior of a disabled button in the browser. However, making those buttons focusable may be good in order to provide context to the user.
